src/components/page.js

```diff
@@ -37,7 +39,15 @@ export default class Page extends PureComponent {
         return (
             <div>
                 <CustomHead />
-                <Nav />
+                <DynamicOnline>
```

Comment:
This component naming is weird and we don't know what it does.
`[Show|Hide]When[Online|Offline]` would probably be easier to read.